docs/pages/dev-guide/developer-guide.md

```diff
@@ -1254,21 +1256,21 @@ pods:
 ```
 
 
-* `EXTERNAL_ROUTES` and `INTERNAL_ROUTES`
+1. Delete these 3 labels from your marathon json:
+    * `DCOS_FRAMEWORK_NAME`
+    * `DCOS_SERVICE_PORT_INDEX`
+    * `DCOS_SERVICE_SCHEME`
+
+1. `EXTERNAL_ROUTES` and `INTERNAL_ROUTES`
```

Comment:
Suggest "1. Add `EXTERNAL_ROUTES` and `INTERNAL_ROUTES` parameters to your service definition.